morphic.js

```diff
@@ -7818,10 +7824,40 @@ MenuMorph.prototype.select = function (aMenuItem) {
     this.selection = aMenuItem;
 };
 
+MenuMorph.prototype.mouseLeave = function () {
+    if (this.parent instanceof MenuMorph) {
+        // this is a submenu so we vanish when lose focus
+		this.destroy();
```

Comment:
hmm... I guess this kinda makes sense. But MacOS leaves the submenu visible as the mouse leaves it. Since submenus are just submorphs (right?) can't we just leave them around until something else gets focus? Of course, I can see how this might lead to situations where a big submenu is in front of the primary one and doesn't let you get back...